src/main/java/com/notably/commons/core/path/AbsolutePath.java

```diff
@@ -1,53 +1,113 @@
 package com.notably.commons.core.path;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
+import java.util.Objects;
 
 import com.notably.commons.core.path.exceptions.InvalidPathException;
 
 /**
  * TODO: Add Javadoc
  */
 public class AbsolutePath implements Path {
+    public static final String INVALIDABSOLUTEPATH = "Invalid absolute path";
+    private final List<String> paths;
+
+
+    public AbsolutePath(String absolutePathString) {
```

Comment:
Let's make this constructor private? Since we're already providing factory static methods like `fromString`, `fromRelativePath`, etc.